charactersheet/charactersheet/templates/dm/encounter_sections/point_of_interest.tmpl.html

```diff
@@ -0,0 +1,189 @@
+<div class="h4">
+  Points of Interest<br />
+  <small>Keep track of all of the interesting locations and
+    objects in the world.</small>
+</div>
+<table class="table table-responsive table-ac-bordered table-hover">
+  <thead>
+    <tr>
+      <th data-bind="click: function(){sortBy('name');}">
+        Name
+        <span data-bind="css: sortArrow('name')"></span>
+      </th>
+      <th data-bind="click: function(){sortBy('description');}">
+        Description
+        <span data-bind="css: sortArrow('description')"></span>
+      </th>
+      <th>
```

Comment:
Pulling the add button to the right? Probably should. It's supposed to be fixed there in all tables imo.